tsconfig.json

```diff
@@ -22,7 +22,8 @@
   },
   "include": [
     "src",
-    "test"
+    "test",
+    "functions/src"
```

Comment:
can we name the "functions" folder "firebaseFunctions" or "databaseFunctions" or "serverlessFunctions", something more specific?

Reply:
renamed